docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -245,4 +245,4 @@ Where to find out more
 ======================
 
 If you'd like to know more about the plans for improving testing Ansible, join the 
-`Ansible community forum <https://forum.ansible.com/>`_
\ No newline at end of file
+`Ansible community forum <https://forum.ansible.com/>`_
```

Comment:
Hmm. Not sure why this line shows up as changed. @MatthiasKunnen Is this unintentional?

Reply:
The file did not have an end-of-file newline. It was automatically added by the editor to comply with standard Linux practice. In short, a text file consists of lines, and lines must end in a newline. Without this newline, programs such as `wc` and `cat` do not work properly.
I checked all rst files in the repo and came to this:
Files terminated with \n: 390
Files not terminated with \n: 63

I can revert this change but it is contrary to standard and the majority of files in the repo. I'll leave the decision up to you.

Reply:
Thanks for the response @MatthiasKunnen No, please don't revert the change. I'm well familiar with the EOF newline convention. It just wasn't clear to me from the diff. Perhaps I should have just looked at the source before I commented. Appreciate the detailed reply as well as the docs improvement. Cheers!